src/main/navigation/navigation_fixedwing.c

```diff
@@ -463,7 +463,23 @@ int16_t applyFixedWingMinSpeedController(timeUs_t currentTimeUs)
 
 int16_t fixedWingPitchToThrottleCorrection(int16_t pitch)
 {
-    return DECIDEGREES_TO_DEGREES(pitch) * navConfig()->fw.pitch_to_throttle;
+    // Calculate base throttle correction from pitch moving average
+    const int16_t movingAverageCycles = navConfig()->fw.pitch_to_throttle_smooth;
+    static int16_t averagePitch = 0;
+
+    averagePitch = (averagePitch * movingAverageCycles + pitch - averagePitch) / movingAverageCycles;
```

Comment:
It's better to use PT1 filter here and controll the cutoff frequency (alpha). Cycles count is susceptible to the rate at which controller is updated.

Reply:
@digitalentity - Thank you for pointing me in the PT1 direction. I'll take a look at it. Do you know if it's possible to filter below 1 Hz?

Reply:
For our PT1 filter implementation the cutoff frequency can be fractional, so no issue with having a sub-Hz filters.

Reply:
@digitalentity I have now implemented a PT1 filter similar to the one for control smoothness.